### ISSUE TITLE
Fail-over not working

### ISSUE THREAD

**Issue by sioux1977** [usecallmanagernz/tftpboot#2]
Hi All, i am trying to configure a fail-over server but seems to be that phone completely ignore the configuration.
I have added a second member into "callManagerGroup"->"members", the first one with priority=0 and the second one with priority=1 (see attached file)
[SEP0023EB52E064.cnf.xml.txt](https://github.com/user-attachments/files/17679519/SEP0023EB52E064.cnf.xml.txt)

If i try to shutdown the first server, the phones stay for one or two minutes without do anything and then auto reboot.

Have you any hint?

**Comment by gareth-palmer** [comment]
You are using UDP as the `<transportLayerProtocol>` , TCP or TLS are the only supported protocols.

Does the phone attempt to register to the secondary server when the primary goes down? Can you provide a SIP trace of the registration attempt.

**Comment by sioux1977** [comment]
Yes the phones try to register to the secondary server in UDP. I have tried in TCP and it works right: phones correctly switch to secondary server when primary goes down. Just another question: do you know if there is a paramter to tell the phone to come back to primary server when it goes up again?
(Regarding packet trasmission using UDP, i have raised t1min in sip.conf to 500ms completely solving the retrasmission issue, in every case, it cause other issue, so it's ok for me to use TCP)

**Comment by gareth-palmer** [comment]
The phone will automatically try and fall back to the primary, it will check the connection to the primary over the course of a few minutes before falling back to prevent flapping between the servers.
